doc/source/development/code_style.rst

```diff
@@ -159,3 +159,29 @@ For example:
 
     # wrong
     from common import test_base
+
+Testing
```

Comment:
>  @simonjayhawkins mentioned that the conventions for urlopen and ZipFile should be captured under our contributing style guide, testing section.

TBH I think that may have been a typo, he was probably copy-and-pasting from his above comment and forgot to change this part, as the two examples you've put below aren't about testing. So, instead of "Testing", I think this section should be called "Misc" (Simon please correct if I'm wrong about this, in which case - my apologies :) )

Reply:
I was thinking the same but wasn't sure. I'll update it Misc once @simonjayhawkins confirms.

Reply:
Misc is fine. testing was incorrect.